rfcs/readme.md

```diff
@@ -0,0 +1,66 @@
+
+The normal GitHub pull request workflow will be used to provide bug fixes and documentation improvements to existing components, however creating new components or significantly modifying existing ones by creating variations will need to have more thorough review/discussion.
+
+This RFC (request for comments) process will allow anyone to submit issues using the provided template to begin discussions and provide the ability for new components to make their way into the design system.
+
+## When to follow this process
+
+It is necessary to follow the RFC process when you are intend to create any significant changes to the OpenUI Design System. Examples of these types of changes are:
+
+* A change which provides significantly alters existing functionality, provides new functionality, or a new primative component.
+* The removal of an existing feature or functionality
+
+Another way to look at it is, if your suggested change will require a new semantic version, or breaking change to an existing feature, it will need to go through the RFC process.
+
+This RFC process is a great way to get involved in the community, suggest new ideas, and discuss them with the community and contributors. It can also be started prior to any design or functionality has been finalized.
+
+Changes that **do not** require an RFC are:
+* Changes in documentation
+* Rephrasing, reorganizing, or refactoring without any functional changes
+* Additions to existing components which are strictly new improvements, accessibility updates, or are bug fixes.
+
+## What the process is
+
+To get a change into the design system, the RFC will first be merged into the repo under the `/rfcs/active` directory. At that point, work on inclusion of the code and documentation can begin. Once the RFC is accepted it will be moved into the `/rfcs/accepted` directory to provide history on the decision process for each component.
+
+To create a new RFC you will:
+
+* Fork the OpenUI Design System repo - https://github.com/openui/design-system
+* Copy the `/rfcs/0000-template.md` file to `/rfcs/active` and rename it to `0000-proposed-component.md` where "proposed-component" is a descriptive title of your component.
+* Fill out the RFC template with your suggestions. Be sure to take care into filling out each portion of the template. **RFCs that do not conform to the template, demonstrate understanding of the imipact of the update, or are disingenous about the impact or alternatives tend to be poorly received**
+* Submit a pull request. Once it's a pull request, the RFC will receive feedback and discussion from the community. Don't worry about getting feedback, it serves to help make the process better. **Be prepared to update and revise your pull request based on community feedback.**
+* Build conensus around your RFC by engaging with the community and their feedback. RFCs with broad support are much more likely to be accepted than those without any comments.
```

Comment:
how can we ensure the community that is engaged is wide enough, eg that it includes people with disabilities and people from non-English speaking etc?